test/config_parser_SUITE.erl

```diff
@@ -12,6 +12,14 @@
 
 -define(HOST, <<"myhost">>).
 
+-define(add_loc(X), {X, #{line => ?LINE}}).
+
+-define(eqf(Expected, Actual), eq_host_config(Expected, Actual)).
+-define(_eqf(Expected, Actual), ?add_loc(fun() -> ?eqf(Expected, Actual) end)).
+-define(errf(Config), 
+        begin ?err(parse_with_host(Config)), ?err(parse_host_config(Config)) end).
+-define(_errf(Config), ?add_loc(fun() -> ?errf(Config) end)).
+
```

Comment:
I think this requires some explanation and from my experience it is too easy to confuse `eqf` with `_eqf` and have a test that does nothing (seen even in MongooseIM code). For me it does not bring much value, but this might be my subjective point of view.